runFfmpeg.sh

```diff
@@ -42,8 +42,8 @@ else
   fix_bounds=true: \
   bordercolor=$FONT_BORDER_COLOR: borderw=1: fontcolor=$FONT_COLOR: \
   textfile=$STREAM_TEXT_PATH: y=$FONT_MUSIC_Y_POS: x=$FONT_MUSIC_X_POS[out]" \
-  -vcodec h264_omx -x264opts keyint=$KEYINT:min-keyint=$KEYINT:scenecut=-1 \
-  -b:v $VIDEO_BIT_RATE -preset veryfast -pix_fmt yuv420p \
-  -c:a libfdk_aac -b:a $AUDIO_BIT_RATE -bufsize 960k \
-  -ar $AUDIO_SAMPLE_RATE -f flv $STREAM_URL
+  -vcodec h264_omx -x264opts keyint="$KEYINT":min-keyint="$KEYINT":scenecut=-1 \
```

Comment:
Yeah I totally learned the importance of quotes in bash while making this :p

Reply:
hahahh 👍